Makefile.toml

```diff
@@ -98,7 +98,7 @@ dependencies = [
 description = "Build all examples as WASM into the target directory"
 category = "ICU4X WASM"
 install_crate = { rustup_component_name = "rust-src" }
-toolchain = "nightly"
+toolchain = "nightly-2021-02-28"
```

Comment:
This is a recent nightly build that is compatible with WABT tooling.  The current nightly outputs the following errors:

```
# wasm-opt
unknown name subsection at 47878
unknown name subsection at 47898

# twiggy
error: WASM error: Invalid name type (at offset 47876)
  caused by: Invalid name type (at offset 47876)
```